x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/eql/trial_license_complete_tier/eql_alert_suppression.ts

```diff
@@ -3181,24 +3179,26 @@ export default ({ getService }: FtrProviderContext) => {
         });
       });
 
-      it('does not suppress alerts outside of duration when query with 3 sequences', async () => {
+      it.only('does not suppress alerts outside of duration when query with 3 sequences', async () => {
         const id = uuidv4();
-        const dateNow = Date.now();
-        const timestampSequenceEvent1 = new Date(dateNow - 5000).toISOString();
-        const timestampSequenceEvent2 = new Date(dateNow - 5500).toISOString();
-        const timestampSequenceEvent3 = new Date(dateNow - 5800).toISOString();
 
-        const firstSequenceEvent = {
-          id,
-          '@timestamp': timestampSequenceEvent1,
-          host: {
-            name: 'host-a',
-          },
-        };
+        const SECOND = 1000;
+        const dateNow = Date.now();
+        const timestamp1 = new Date(dateNow - 50 * SECOND).toISOString();
+        const timestamp2 = new Date(dateNow - 49 * SECOND).toISOString();
+        const timestamp3 = new Date(dateNow - 48 * SECOND).toISOString();
+        const timestamp4 = new Date(dateNow - 5 * SECOND).toISOString();
+        const timestamp5 = new Date(dateNow - 4 * SECOND).toISOString();
+        const timestamp6 = new Date(dateNow - 3 * SECOND).toISOString();
+
+        const seqDoc = (ts: string) => ({ id, '@timestamp': ts, host: { name: 'host-a' } });
```

Comment:
Nit: you could call `.toISOString()` in the helper, instead of above:

```suggestion
        const timestamp1 = new Date(dateNow - 50 * SECOND);
        const timestamp2 = new Date(dateNow - 49 * SECOND);
        const timestamp3 = new Date(dateNow - 48 * SECOND);
        const timestamp4 = new Date(dateNow - 5 * SECOND);
        const timestamp5 = new Date(dateNow - 4 * SECOND);
        const timestamp6 = new Date(dateNow - 3 * SECOND);

        const buildSequenceDoc = (ts: Date) => ({ id, '@timestamp': ts.toISOString(), host: { name: 'host-a' } });
```